CyclePhillyAndroid/build.gradle

```diff
@@ -20,19 +20,48 @@ repositories {
 android {
     compileSdkVersion 21
     buildToolsVersion '21.1.2'
+
     defaultConfig {
         minSdkVersion 10
         targetSdkVersion 21
     }
+
     productFlavors {
+        mainActivity {
+            manifestPlaceholders = [mainCategory: "android.intent.category.LAUNCHER"]
+        }
+        mainInput {
```

Comment:
This lets you still run the original Launcher Activity: Main Input as long as you go into Run > Edit Configurations and change the launcher activity to MainInput.

![image](https://cloud.githubusercontent.com/assets/2046641/9373175/633e48e2-46b5-11e5-812a-91c61b308b68.png)

![image](https://cloud.githubusercontent.com/assets/2046641/9373191/86fab590-46b5-11e5-8854-a5556d811d0d.png)


You can switch between MainInput and MainActivity product flavors in the build variants:
![image](https://cloud.githubusercontent.com/assets/2046641/9373198/976bf5c4-46b5-11e5-8f57-2599346ed4ab.png)